main.tf

```
@@ -64,6 +64,7 @@ resource "aws_redshift_cluster" "this" {
   master_password                   = var.snapshot_identifier == null && !var.manage_master_password ? local.master_password : null
   master_password_secret_kms_key_id = var.master_password_secret_kms_key_id
   master_username                   = var.master_username
+  multi_az                          = var.number_of_nodes > 1 ? var.multi_az : null
```

```suggestion
  multi_az                          = var.multi_az
```

We should not do logic like this in the modules and let users define the values they want (even if it doesn't make sense to have multi_az with one node).